cpp/include/cudf/copying.hpp

```diff
@@ -344,6 +344,14 @@ std::vector<gdf_column*> split(gdf_column const &          input_column,
                                gdf_index_type const*      indices,
                                gdf_size_type              num_indices);
 
+
+
+gdf_column shift(
```

Comment:
Looks like you got rid of the `table` API in favor of the `column` API. I was suggesting the opposite.

Reply:
just moved the existing code over to 'copying'. will swap out the types / names soon.